emily/handler/src/api/models/withdrawal/mod.rs

```diff
@@ -99,23 +96,23 @@ pub struct WithdrawalParameters {
 #[serde(rename_all = "camelCase")]
 pub struct WithdrawalInfo {
     /// The id of the Stacks withdrawal request that initiated the sBTC operation.
-    pub request_id: WithdrawalId,
+    pub request_id: u64,
     /// The stacks block hash in which this request id was initiated.
-    pub stacks_block_hash: StacksBlockHash,
+    pub stacks_block_hash: String,
     /// The height of the Stacks block in which this request id was initiated.
-    pub stacks_block_height: BlockHeight,
+    pub stacks_block_height: u64,
     /// The recipient Bitcoin address.
-    pub recipient: BitcoinAddress,
+    pub recipient: String,
     /// Amount of BTC being withdrawn.
-    pub amount: Satoshis,
+    pub amount: u64,
```

Comment:
I would update the comment to state that this is in sats, although maybe that is obvious given that the type is `u64`?

Reply:
I agree it's not obvious. I'll fix it

Reply:
Changed the comment above.

```rust
    /// Amount of BTC being withdrawn in satoshis.
    pub amount: u64,
```